src/test/regress/expected/multi_prepare_sql.out

```diff
@@ -1195,6 +1195,31 @@ EXECUTE countsome; -- no replanning
 ---------------------------------------------------------------------
 (0 rows)
 
+-- Standard planner converted text and varchar casts to cstring in some cases
+-- We make sure we convert it back to text when parsing the expression
+-- https://github.com/citusdata/citus/issues/6061
+-- https://github.com/citusdata/citus/issues/5646
+-- https://github.com/citusdata/citus/issues/5033
+CREATE TABLE test(t timestamptz, user_id int);
+SELECT create_distributed_table('test', 'user_id');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+PREPARE test_statement(text) AS
```

Comment:
I'd put this test to arbitrary configs tests, which should cover a lot more tests. Can we move these to https://github.com/citusdata/citus/blob/main/src/test/regress/sql/prepared_statements_4.sql or such?

Please remember that you should create the tables in `prepared_statements_create_load ` to use in prepared_statements_4``